src/clusto/services/http.py

```diff
@@ -201,12 +253,55 @@ def show(self, request, prefetch_attrs=None):
         for x in self.obj.attrs():
             attrs.append(unclusto(x))
         result['attrs'] = attrs
-        result['contents'] = [unclusto(x, prefetch_attrs) for x in self.obj.contents()]
-        result['parents'] = [unclusto(x, prefetch_attrs) for x in self.obj.parents()]
+        contents = self.obj.contents()
+        parents = self.obj.parents()
+
+        result['contents'] = mass_unclusto(contents, prefetch_attrs)
+        result['parents'] = mass_unclusto(parents, prefetch_attrs)
```

Comment:
nit: I feel like in general, we should probably fall back to the old way if we are less likely to break existing stuff